test/integration-test/go.mod

```diff
@@ -2,9 +2,11 @@ module github.com/dell/csi-volumegroup-snapshotter/test/integration-test
 
 //replace github.com/dell/dell-csi-extensions/volumeGroupSnapshot => ../../dell-csi-extensions/volumeGroupSnapshot
 
+replace github.com/dell/gofsutil => ../../../gofsutil
+
 replace github.com/dell/csi-volumegroup-snapshotter => ../../
 
-//replace github.com/dell/vxflexos => ./csi-powerflex
+replace github.com/dell/csi-vxflexos/v2 => ../../../csi-powerflex
```

Comment:
This will work, but can we change it to:
```replace github.com/dell/vxflexos => ./csi-powerflex```
Reason: our Jenkins tests expect csi-powerflex to be inside the int test directory:
```cd /root/${DRIVER}/test/integration-test && git clone https://github.com/dell/csi-powerflex```

Reply:
We could do that, although what's the reasoning for having the driver inside the int test directory? Wouldn't it make more sense to expect it to be in the same place as the driver repo?

Reply:
@JacobGros did revert it as per your request tho

Reply:
keeping it inside int directory minimizes the risk of collisions. For example, if a customer had a different version of csi-powerflex cloned to a dir, and inside this dir also had vg-snapshotter; they would have to move their csi-powerflex and clone the newer version for their int tests to run. This works, but it's a little invasive. Keeping the repo in the int-test dir ensures that the customers file structure outside of our repo won't matter.